components/search/SearchResults.vue

```diff
@@ -69,9 +67,14 @@
     },
 
     methods: {
-      cardTexts(result) {
+      cardTexts(result, variant) {
         const texts = [result.edmDataProvider];
         if (result.dcCreator) texts.unshift(result.dcCreator);
+
+        if (variant === 'list') {
+          if (result.dcDescription) texts.unshift(result.dcDescription);
```

Comment:
This needs to only be added if dcTitle is present, else it will end up duplicated from being used instead of the title.

To see this, search for `proxy_dc_description:* NOT proxy_dc_title:*`